### PR TITLE
chore: Delete VueBus

### PR DESCRIPTION
**Pull request by salisdemos**
Remove / Refactor the last ocurences of the Vue Bus to get rid of that bloddy bastard.... finally

I hope I haven't overseen anything. I tried to test it locally and it looked good to me. But we have to test this really good since it was baked in quite nasty. I am not sure what that could break

related https://github.com/demos-europe/demosplan-project-teilhabe/pull/21

### CODE REVIEW

File: client/js/InitVue.js [hunk 1]
```diff
@@ -51,6 +50,9 @@ function initialize (components = {}, storeModules = {}, apiStoreModules = [], p
 
         const mountedEvent = new Event('vue-mounted')
         document.dispatchEvent(mountedEvent)
+        setTimeout(() => {
+          window.mounted = true
+        }, 5)
```
Comment (spiess-demos) [inline client/js/InitVue.js:55]: Why is this necessary?

File: client/js/components/procedure/basicSettings/DpProcedureCoordinate.vue [hunk 2]
```diff
@@ -244,17 +244,26 @@ export default {
...
-    window.Bus.on('wizard:show toggleAnything:clicked', (sender, data) => {
-      //  Only fire when relevant wizard step / toggleAnything toggleId is transmitted
+    const handleWizardShow = data => {
       if (data === Translator.trans('wizard.topic.location') || data === 'procedureLocation') {
         this.$refs.map.updateMapInstance()
       }
+    }
+    //  Listeners are added because the OpenLayers map needs to be initialized on a visible element
+    document.addEventListener('wizard:show', ({ data }) => {
+      //  Only fire when relevant wizard step / toggleAnything toggleId is transmitted
```
Comment (spiess-demos) [inline client/js/components/procedure/basicSettings/DpProcedureCoordinate.vue:254]:
```suggestion
      //  Only fire when relevant wizard step is transmitted
```

**Review by spiess-demos — COMMENTED** [review]
Apart from some nitpicky suggestions and a general question, the changes look reasonable to me. 
The changes in DpPublicLayerListLayer.vue and MapTools.vue may change the behavior of the public map layer controls, as they emit events that are actually listened too (before, wvents were simply emitted the wrong way). We will have to pay attention primarily to the public layer list when testing.

**Review by spiess-demos — APPROVED** [review]
lgtm!

File: client/js/InitVue.js (hunk 1 above)
Comment (hwiem) [reply client/js/InitVue.js:55]: Wouldn't it be a better solution to set the variable in an event listener, like so:

```
document.addEventListener('vue-mounted', () => {
  window.mounted = true
}
```
